code/src/App.js

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+/* eslint-disable indent */
+/* eslint-disable arrow-parens */
+/* eslint-disable react/jsx-closing-tag-location */
+/* eslint-disable max-len */
+/* eslint-disable react/jsx-no-useless-fragment */
+/* eslint-disable jsx-quotes */
+import React from 'react'
+import AlbumArtist from 'components/AlbumArtist'
+import Header from 'components/Header'
+import Playlists from 'components/Playlists';
+import playlists from './stretch-goal.json';
 import data from './data.json';
+import Artwork from './components/Artwork'
+import AlbumName from './components/AlbumName'
 
-console.log(data);
+import './index.css';
 
-export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
+export const App = () => (
+  <div className='body-container'>
+    <Header />
+    <div className='music-section'>
+    <div className='music-container'>
+      {data.albums.items.map(item => {
+        return <>
+          <div className='Albums'>
+            <Artwork key={item.id} item={item} img={item.images} />
+            <AlbumName key={item.name} AlbumName={item.name} AlbumUrl={item.external_urls.spotify} />
+            <AlbumArtist key={item.artists.name} AlbumArtist={item} />
+          </div>
+        </>;
+      })}
```

Comment:
the App itself is very clean and the structure of the components is clear and well thought-out. well done!